src/Pages/Login/Login.js

```diff
@@ -1,83 +1,125 @@
+    const password = formData.get('password');
+
+    try {
+      const loginStatus = await loginUser(email, password);
+      if (!loginStatus.error) {
+        setAuthenticated(true);
+        window.localStorage.setItem('jwtToken', loginStatus.token);
+        if (queryParams.get('redirect')) {
+          window.location.href = queryParams.get('redirect');
+        } else {
+          window.location.reload();
+        }
       } else {
-        window.location.reload();
-      }
-    } else {
-      if(loginStatus.responseData === undefined || loginStatus.responseData.data.message === undefined){
-        setErrorMsg('Backend May Be down, check with dev team!');
-      }else{
-        setErrorMsg(
-          loginStatus.responseData && loginStatus.responseData.data.message
-        );
+        if (loginStatus.responseData === undefined || loginStatus.responseData.data.message === undefined) {
+          setErrorMsg('Backend May Be down, check with dev team!');
+        } else {
+          setErrorMsg(loginStatus.responseData && loginStatus.responseData.data.message);
+        }
       }
+    } catch (error) {
+      setErrorMsg('An unexpected error occurred. Please try again.');
+    } finally {
+      setIsLoading(false);
     }
   }
 
   return (
-    <div className = 'flex-none md:flex'>
-      <div className='rounded-3xl backdrop-blur-sm shadow-2xl md:w-1/3  mt-20 pb-4 mb-auto ml-auto mr-auto px-5 text-center items-center justify-center'>
-        <div className='flex justify-center'>
-          <img id='img' alt='sce logo' src={`${window.location.origin}/images/SCE-glow.webp`} width='2rem'  className='w-2/3 px-auto'/>
-        </div>
-        <form onSubmit={handleSubmit} className=''>
-          <div className='flex flex-col items-center'>
-            <label className="form-control w-full max-w-xs">
-              <div className="label">
-                <span className="label-text">Email</span>
+    <div className={`flex flex-col gap-6 ${className || ''}`} {...props}>
+      <div className="card bg-base-100 border border-white/10">
+        <div className="card-body">
+          <div className="text-center mb-6">
+            <h2 className="card-title text-2xl font-bold justify-center">Login to your account</h2>
+            <p className="text-base-content/70 text-sm mt-2">
+              Enter your email below to login to your account
+            </p>
+          </div>
+          <form onSubmit={handleSubmit}>
+            <div className="flex flex-col gap-6">
+              <div className="grid gap-3">
+                <label htmlFor="email" className="label">
+                  <span className="label-text text-sm font-medium">Email</span>
+                </label>
+                <input
+                  id="email"
+                  name="email"
+                  type="email"
+                  placeholder="m@example.com"
+                  required
+                  className="input input-bordered w-full"
```

Comment:
does it have to be a form, we could easily do
`onChange={(e) => setEmail(e.target.value)}`

this is what was there before